packages/client/src/components/MovieList/MovieList.jsx

```diff
@@ -0,0 +1,197 @@
+      image_location:
+        'https://www.themoviedb.org/t/p/w1280/nts4iOmNnq7GNicycMJ9pSAn204.jpg',
+      price: 90,
+    },
+    {
+      id: 6,
+      category_id: 1,
+      title: 'Legend',
+      description:
+        'Suave, charming and volatile, Reggie Kray and his unstable twin brother Ronnie start to leave their mark on the London underworld in the 1960s. Using violence to get what they want, the siblings orchestrate robberies and murders.',
+      movie_year: 2015,
+      image_location:
+        'https://www.themoviedb.org/t/p/w1280/4shf5Alq4KWCKqrAAQe0JGJHYp5.jpg',
+      price: 66,
+    },
+    {
+      id: 7,
+      category_id: 4,
+      title: 'Stuart Little',
+      description:
+        'The adventures of a heroic and debonair stalwart mouse named Stuart Little with human qualities, who faces some comic misadventures while searching for his lost bird friend and living with a human family as their child.',
+      movie_year: 1999,
+      image_location:
+        'https://www.themoviedb.org/t/p/w1280/jwydlxxLPVxM5uCfJybEsANzvvg.jpg',
+      price: 100,
+    },
+    {
+      id: 8,
+      category_id: 2,
+      title: 'Gone Girl',
+      description:
+        'With his wife his disappearance having become the focus of an intense media circus, a man sees the spotlight turned on him when it is suspected that he may not be innocent.',
+      movie_year: 2014,
+      image_location:
+        'https://www.themoviedb.org/t/p/w1280/qymaJhucquUwjpb8oiqynMeXnID.jpg',
+      price: 120,
+    },
+    {
+      id: 9,
+      category_id: 5,
+      title: 'The Exorcist',
+      description:
+        "Ophelia, a young nun recently arriving in the town of San Ramon, is forced to perform an exorcism on a pregnant woman in danger of dying. Just when she thinks her possession has ended, she discovers that the evil presence hasn't disappeared yet",
+      movie_year: 2014,
+      image_location:
+        'https://www.themoviedb.org/t/p/w1280/d07xtqwq1uriQ1hda6qeu8Skt5m.jpg',
+      price: 666,
+    },
+    {
+      id: 10,
+      category_id: 3,
+      title: 'The Dark Knight',
+      description:
+        'Batman raises the stakes in his war on crime. With the help of Lt. Jim Gordon and District Attorney Harvey Dent, Batman sets out to dismantle the remaining criminal organizations that plague the streets. The partnership proves to be effective.',
+      movie_year: 2008,
+      image_location:
+        'https://www.themoviedb.org/t/p/w1280/z4XwvnDZSm40PCdg9vFJmo0W9ri.jpg',
+      price: 120,
+    },
+  ];
+  return (
+    <Box bgcolor="mainGreen" fontFamily="Inter" marginTop="150rem">
+      <Typography
+        sx={{
+          position: 'relative',
+          height: '1.87rem',
+          width: '17.81rem',
+          top: '1.56rem',
+          left: '35rem',
+          fontFamily: 'Inter',
+          fontStyle: 'normal',
+          fontWeight: 'bold',
+          fontSize: '1.56rem',
+          color: '#003C2D',
+        }}
```

Comment:
everything here is correct but I would suggest that you guys have discussion in the team(s) to create some small iternal component library, because I see a lot of repetitions in styling things across PRs :)

Reply:
👍🏻

Reply:
Hi @magdazelena I think, we can merge this, and refactor the components later once we meet on Sunday with the team

Reply:
Hi @shobanamg I am working on the url part as @magdazelena suggested once this will fix and she approves I will merge 🙏🏾.